backend/src/test/java/gov/cdc/usds/simplereport/idp/authentication/LiveOktaAuthenticationTest.java

```diff
@@ -0,0 +1,517 @@
+package gov.cdc.usds.simplereport.idp.authentication;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import dev.samstevens.totp.code.CodeGenerator;
+import dev.samstevens.totp.code.DefaultCodeGenerator;
+import gov.cdc.usds.simplereport.api.BaseFullStackTest;
+import gov.cdc.usds.simplereport.api.model.errors.InvalidActivationLinkException;
+import gov.cdc.usds.simplereport.api.model.errors.OktaAuthenticationFailureException;
+import gov.cdc.usds.simplereport.api.model.useraccountcreation.FactorAndQrCode;
+import java.time.Instant;
+import java.util.List;
+import org.json.JSONObject;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * WARNING: THIS TEST CREATES A REAL USER IN OKTA!
+ *
+ * <p>This tests LiveOktaAuthentication by creating a user and sending them through the account
+ * creation flow. The user is not associated with an organization, and is deleted at the end of the
+ * test. Running this test does add to our Okta quota, so plase run sparingly.
+ */
+@TestInstance(Lifecycle.PER_CLASS)
+@TestMethodOrder(OrderAnnotation.class)
+class LiveOktaAuthenticationTest extends BaseFullStackTest {
+
+  private static final String PHONE_NUMBER = "999-999-9999";
+  private static final String FORMATTED_PHONE_NUMBER = "+19999999999";
+  private static final String EMAIL = "test@example.com";
```

Comment:
If we do come up with a way to run these automatically, the fixed email address will prevent it from being run successfully from multiple clients concurrently.

Reply:
Hmmm, that's a good point. I wonder if it'd be worth doing something like 

```
String now = Instant.now().toString();
String EMAIL = "test+" + now + "@example.com"
```

which would make some ugly email addresses, but would hopefully be unique enough that it could run from multiple clients.